environment.yml

```diff
@@ -18,6 +18,7 @@ dependencies:
   - joblib~=1.2.0
   - toml
   - protobuf=3.20
+  - protozfits=2.2
```

Comment:
In `setup.py` you define `~=2.2` (Which means any 2.x that is greater equal 2.2), and here you define `=2.2`, which means "the latest 2.2.x".

I.e. installing via pip will currently give you 2.4, but setting up the env will give you 2.2

Reply:
yes, I misread the comment. 

> What is better here? Should I be afraid of any existing or future 2.x being incompatible with anything I do with protozfits? I just checked I have been testing with 2.3.0

Are the 2.x minor releases and not supposed to break anything? In any case maybe safer to limit <2.5?

Reply:
So the equivalent for what is in `setup.py` would be `>=2.2,<3`